### ISSUE TITLE
New lint: Use actually getters instead of getter functions (or vice versa?)

### ISSUE THREAD

**Issue by danylo-safonov-solid** [solid-software/solid_lints#150]
This will allow projects to enforce usage of just a single style:

```dart
 X get x ...;
```

```dart
X getX() ...
```
